src/scss/_arrow.scss

```diff
@@ -34,6 +49,10 @@
 	}
 }
 
+/// Top arrow styling
+///
+/// @param {Color} color
```

Comment:
Also, default values should be supplied as such:

`@param {type} $name (default value) - description`

In that case not sure how it would be written if the default requires o-colors.